File: app/backend/models/tipo_notificacion_model.py
from sqlalchemy import Column, Integer, String, Text, Date, DateTime
from backend.database.connection import Base

class TipoNotificacion(Base):
    __tablename__ = "TipoNotificacion"

    IdTipoNotificacion = Column(Integer, primary_key=True, index=True, autoincrement=False)
    Descripcion = Column(String(50), nullable=False)
    DescCorta = Column(String(15), nullable=True)
    AudFecha = Column(DateTime, nullable=True)
    AudUsuario = Column(Integer, nullable=True)

File: app/frontend/src/app/features/tipos-notificacion/models/tipo-notificacion.model.ts
export interface TipoNotificacion {
  IdTipoNotificacion: number;
  Descripcion: string;
  DescCorta?: string;
  AudUsuario?: number;
  AudFecha?: string;
}

export interface TipoNotificacionCreate {
  Descripcion: string;
  DescCorta?: string;
}